streaming/multimodal/webvid.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
I'm not sure this file is worth merging, given it has `# Processing goes here.`? Maybe at least raise a not implemented error or something?

Reply:
this class included as a demonstration of where to modify in order to do your own on-the-fly sample processing